src/utils.ts

```diff
@@ -247,13 +248,28 @@ export default abstract class BaseCommand extends Command {
     return parsed
   }
 
-  protected async initDB (path: string, sync = false): Promise<void> {
+  protected async initDB (path: string, options?: CliInitDbOptions & { forcePrompt?: boolean }): Promise<void> {
     const sequelize = await sequelizeFactory(path)
+    const migrator = DbMigration.getInstance(sequelize)
 
-    if (sync) {
+    if (options?.sync) {
       await sequelize.sync({ force: true })
     }
+
+    // Init store
     await initStore(sequelize)
+
+    // Run migration
+    if (options?.migrate) {
+      if ((await migrator.pending()).length) {
+        if (options?.forcePrompt || await this.prompt('DB Migration required! Run Migration (y/n)?')) {
```

Comment:
Hmm is `forcePrompt` used somewhere else? Or just here? The name seems bit confusing for me, like I don't really understand what is its purpose here 😅

Reply:
This is used in test's and `init` command

Reply:
Oh, got it. Yeah, I like that one more.